packages/victory-core/src/victory-accessibility-group/victory-accessibility-group.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class VictoryAccessibilityGroup extends React.Component {
+  static displayName = "VictoryAccessibilityGroup";
+  static role = "container";
+  static propTypes = {
+    "aria-describedby": PropTypes.string,
+    "aria-label": PropTypes.string,
+    desc: PropTypes.string,
+    descId: PropTypes.string,
+    children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
+    className: PropTypes.string,
+    tabIndex: PropTypes.number
+  };
+
+  static defaultProps = {
+    className: "VictoryAccessibilityGroup"
+  };
+
+  render() {
+    const { desc, children, className, descId, tabIndex } = this.props;
+
+    return desc ? (
+      <g aria-label={this.props["aria-label"]} className={className} tabIndex={tabIndex}>
+        <desc aria-describedby={this.props["aria-describedby"]} id={descId}>
```

Comment:
Also, when there is a `desc`, there needs to be a back up for if the user doesn't provide a `descId`

Reply:
What about enforcing this with types?
No error if `desc` is not provided (will just render the `g` w/props)
![fine](https://user-images.githubusercontent.com/27159818/95268255-2fc84200-07f4-11eb-8217-c72e961147f1.png)

Then if `desc` is present, require `descId`
![fine](https://user-images.githubusercontent.com/27159818/95268312-50909780-07f4-11eb-8abb-6d8b46ecd71f.png)

I'm sure theres a more efficient way to go about this, but basically:
```

export type VictoryAccessibleGroupProps = {
  desc: string;
  descId: string;
  "aria-describedby"?: string;
  "aria-label": string;
  children?: React.ReactElement | React.ReactElement[];
  className?: string;
  tabIndex?: number;
} | {
  desc?: false;
  descId?: never;
  "aria-describedby"?: string;
  "aria-label": string;
  children?: React.ReactElement | React.ReactElement[];
  className?: string;
  tabIndex?: number;
}
```

Reply:
OK found a better resource to understand the described by attribute, https://developer.paciellogroup.com/blog/2018/09/describing-aria-describedby/

in this article, `aria-describedby` and `descId` should be identical, with `aria-describedby` using the `descId` (as you had said)
I am wondering if `aria-describedby` should even be a prop the user controls: 
```
if(descId) => descId is required, aria-describedby will use the same input
else arial-label should suffice
```
![error](https://user-images.githubusercontent.com/27159818/95352251-4b2c5f00-087f-11eb-8bc2-ed5c6bd3d1e7.png)

Reply:
Hmm... I think `aria-describedby` should still be a prop in the case where a user wants to describe the group with some other, non-victory element. If they provide both a `desc` prop, though, the user's `aria-describedby` should be used to set the `descId` so that they definitely match. If they provide only a `desc` and no `aria-describedby` we will need to generate an id to use for both `aria-describedby` and `descId`. So, I think `descId` probably shouldn't be a prop.